### ISSUE TITLE
need your help！

### ISSUE THREAD

**Issue by Lj4040** [seanie12/CLAPS#8]
labels = torch.arange(batch_size,
                                  device=input_ids.device)

            cont_loss = cont_crit(logits, labels)
            new_cont_loss = cont_crit(new_logits, labels)
Hello! Look at your work and feel very good! So I tried to learn, in the process of learning I have a puzzle. Why is it always compared with this label in the model? This label is only randomly generated according to the size of batch_size 1, 2, 3, 4... I don't seem to have found the right position for such loss in the paper. Could you please give me an answer? I am very confused about it
